text/0000-sawtooth-governance.md

```diff
@@ -0,0 +1,348 @@
+pointing out the tradeoffs that were raised and explaining how they were
+weighted, and **never introducing new arguments**.
+
+### Initial Subteams
+
+The following is a proposed list of initial subteams which is subject to change
+at the discretion of the root team, once it has formed.
+
+**Core** - Responsible for the design, architecture, performance, stability,
+ and security of the core Sawtooth platform
+
+**Application SDKs** - Responsible for the design and consistency of
+user-facing SDKs for application development
+
+**Consensus** - Responsible for the supported consensus algorithms and
+the consensus interface
+
+**Seth** - Responsible for the Sawtooth integration with Ethereum
+
+**Community Outreach** - Responsible for writing and hosting documentation,
+managing the main Sawtooth website, promoting Sawtooth to the public, managing
+demos, and providing the community with training and support
+
+**Release Management** - Responsible for release related issues such as
+dependency management, license compliance, version control, and upgrade and
+backwards compatibility
+
+**Continuous Integration** - Responsible for test and build environments and
+deployment artifacts
+
+## Contributor Permission Levels
+
+The Sawtooth community encourages contributions to the project from all
+interested individuals. The project defines three levels of permissions that
+determine what actions a contributor can perform on a particular repository.
+
+A **reviewer** has "read" permission, meaning they can review pull requests.
+
+A **committer** has "write" permission, meaning they can merge pull requests
+once they have been approved.
+
+Any community member can be promoted to a reviewer or committer by a
+maintainer. Reviewers and committers lose their status when it is removed by a
+maintainer. This is usually due to inactivity, but can also be used for
+moderation.
+
+A **maintainer** has permission to approve changes. All pull-requests must be
+approved by at least 2 maintainers before a committer may merge it.
+Maintainers are expected to:
+
+- Carefully review pull requests and leave thoughtful, constructive feedback
+- Ensure the component is up-to-date with patches and security updates
+- Participate in community discussions related to their component
+
+A committer can be promoted to a maintainer by unanimous vote by the existing
+maintainers. Maintainers lose their status due to:
+
+- Inactivity of 12 Months
+- Unanimous vote by all other Maintainers
+
+## Code of Conduct
+
+Members of the Sawtooth community are expected to abide by the [Hyperledger Code of Conduct][hyperledger-coc].
+Violations of this code of conduct by any community member may result in
+disciplinary action at the discretion of the root team (by unanimous vote,
+excluding the offending member if applicable).
+
+[hyperledger-coc]: https://wiki.hyperledger.org/community/hyperledger-project-code-of-conduct
+
+# Drawbacks
+[drawbacks]: #drawbacks
```

Comment:
This doesn't specify anything about e.g. release processes and strategies, or who can approve or veto a release of the software. I think this is the most important part, since the project is primarily about producing releases of the Sawtooth software.